templates/zerver/help/view-someones-profile.md

```diff
@@ -10,21 +10,16 @@ streams for which [you can see all subscribers](/help/stream-permissions).
 
 ## View someone's profile
 
-### Via the message list
-
 {start_tabs}
 
-1. Click on a user's profile picture.
-
-1. Click **View full profile**.
-
-{end_tabs}
+{tab|desktop-web}
 
-### Via the right sidebar
+{!right-sidebar-view-full-profile.md!}
 
-{start_tabs}
+!!! Tip ""
 
-{!right-sidebar-view-full-profile.md!}
+    You can also click on a user's profile picture or name and then
```

Comment:
Maybe "... picture or name in the message list", since the reference to the message list was previously contained in the heading?

Reply:
How about something like this for the tip? I think it's helpful to be a bit more specific.

> You can also click on a user's profile picture or name on a message they sent, and skip to the last step.

I like the bottom placement of the tip, since it makes it easier to follow the instructions without getting distracted by it if you don't need it.